ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -2,7 +2,7 @@ periodics:
 - agent: kubernetes
   cluster: build01
   decorate: true
-  interval: 2h
+  interval: '@yearly'
```

Comment:
`@yearly` is a `cron` value, not an interval

Reply:
Updated. Thanks @petr-muller .